src/python_bindings/ind/bind_ind.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+namespace python_bindings {
+void BindInd(pybind11::module_& main_module);
+}
```

Comment:
I wonder why the formatter didn't catch this
```suggestion
}  // namespace python_bindings
```